.github/workflows/rails_ci.yml

```diff
@@ -5,37 +5,43 @@
 # This workflow will install a prebuilt Ruby version, install dependencies, and
 # run tests and linters.
 name: "Ruby on Rails CI"
-on:
-  push:
-    branches: [ $default-branch, 2024-upgrades-main ]
-  pull_request:
-    branches: [ $default-branch, 2024-upgrades-main ]
+on: push
```

Comment:
this is intended to be temporary, to allow me to test this out. though I think running on every push would be useful - I was finding that pushes to an opened PR weren't triggering a re-run